virtualmachine/vsphere/util.go

```diff
@@ -558,13 +560,20 @@ func removeExistingNetworks(vm *VM, vmObj *object.VirtualMachine) ([]types.BaseV
 }
 
 var cloneFromTemplate = func(vm *VM, dcMo *mo.Datacenter, usableDatastores []string) error {
```

Comment:
Not related to your change, but did not understand why we have functions like these here?

Reply:
This is how these functions are created in libretto when libretto was forked to apporbit. Don't know the significance of this here.